pkg/tso/keyspace_group_manager.go

```diff
@@ -446,6 +447,10 @@ func (kgm *KeyspaceGroupManager) updateKeyspaceGroup(group *endpoint.KeyspaceGro
 		if kgm.ams[group.ID].Load() != nil {
 			log.Info("keyspace group already initialized, so update meta only",
 				zap.Uint32("keyspace-group-id", group.ID))
+			group.KeyspaceLookupTable = make(map[uint32]struct{})
+			for _, kid := range group.Keyspaces {
+				group.KeyspaceLookupTable[kid] = struct{}{}
```

Comment:
Do we need to clean up the keyspaces that may not be in this group anymore in this situation?